Applications/Telemetry/telemetry.py

```diff
@@ -129,30 +174,23 @@ def dbTelemetry():
         timespan = request.args.get("timespan", 5*60)
         limit = request.args.get("limit")
 
-        nodeid = str(nodeid)
-        direction = int(direction)
-        callsign = str(callsign).upper()
-        timespan = int(timespan)
-        if limit != None:
-            limit = int(limit)
-
-    except ValueError as e:
-        logger.error("ValueError: " + str(e))
-        return json.dumps({"error": str(e)}), 400
-    except IndexError as e:
-        logger.error("IndexError: " + str(e))
-        return json.dumps({"error": str(e)}), 400
-    except KeyError as e:
-        logger.error("KeyError: " + str(e))
+    except IOError as e:
+        logger.error("IOError: " + str(e))
         return json.dumps({"error": str(e)}), 400
 
+    nodeid = str(nodeid)
+    direction = int(direction)
+    callsign = str(callsign).upper()
+    timespan = int(timespan)
+    if limit != None:
+        limit = int(limit)
+
     # Validate timespan
     if timespan <= 0:
         message = "Error: Timespan '{0}' is invalid".format(timespan)
         return json.dumps({"error": message}), 400
 
     # Create tuple of parameters for SQLite3
-    parameters = {}
```

Comment:
Assumes local function variables are lost after runtime and reset to default nothing.